### ISSUE TITLE
PDF output

### ISSUE THREAD

**Issue by bradfrost** [stubbornella/type-o-matic#10]
JSON export is a great feature, but I think printing the output as a PDF would be awesome too. Non-developer types (designers, business owners, clients) would be able to visually see how all over the board their type is by examining the sample text.

**Comment by 0xjjpa** [comment]
@bradfrost I agree that it would be useful. Shouldn't be to hard to make the output open in a external window where then users can print it.

**Comment by stubbornella** [comment]
My team wrote a tool that takes the JSON data and makes a nice report out of it. Maybe we an incorporate that here?

**Comment by 0xjjpa** [comment]
@stubbornella Nice. I'm abstracting most of the tool from the extension for the Chrome version. I'll point you to the repository when I'm done with the tests.

**Comment by pzi** [comment]
Sounds like a great feature to have. Any development update @jjperezaguinaga ? :)

**Comment by dennisfrank** [comment]
If you are on a Mac you can select all (CMD+A) and copy (CMD+C) Type-o-matic’s results table and paste it into TextEdit.app. Select all again and copy and paste into a Numbers.app table. Now you can sort, filter, print the results or export to PDF. Works really great for me.

![typo-o-matic numbers](https://cloud.githubusercontent.com/assets/65968/5348232/589a60aa-7f2e-11e4-9d0b-407fd906fc1a.png)
